lib/virt_autotest/virtual_network_utils.pm

```diff
@@ -357,10 +357,12 @@ sub clean_all_virt_networks {
     foreach my $vnet (split(/\n+/, $_virt_networks)) {
         my $_br = script_output(q@virsh net-dumpxml @ . $vnet . q@|grep -o "bridge name=[^\s]*" | sed  's#bridge name=##'@, type_command => 0, proceed_on_failure => 0);
         script_run("virsh net-destroy $vnet");
-        assert_script_run("virsh net-undefine $vnet");
+        script_run("virsh net-undefine $vnet");
         assert_script_run("if ip a|grep $_br;then ip link del $_br;fi");
         save_screenshot;
     }
+
+    die "Virtual networks are not fully cleaned!" if (script_output("virsh net-list --name --all"));
```

Comment:
Necessary to remove "default" as well ? I do not think so.

Reply:
This function is "sub clean_all_virt_netoworks".

Reply:
I think you'd better to keep "default". You can remove all the others. If any following tests to run after this and they use "default", there is nearly no way to re-create it.

Reply:
Additionally it looks like deleting br0 on host, although deleting "default" would not lead to failures directly, but it would be better to keep it. If "default" is gone, there is no way to know what it is afterwards, either from its name or other things. The subroutine name still apply, no need to delete "default", the built-in one.

Reply:
The default network on alp can be seen in http://10.67.129.185/tests/537#step/setup_kvm_container/147. So you can see, on baremetal alp host, no default bridge br0 at all. Within libvirtd container, the "default" network is sle_network, a nat based network which has nothing to do with br0 either. Even if we create br0 during host preparation but before libvirtd container is created, the br0 is not hurt at all. 
This function cleans VIRTUAL networks, rather than physical networks. So the physical networks on host are still there.
Removing the "default" virtual network won't hurt the libvirtd container connectivity either, and it will exclude any potential impact to some of our hard-coded existing code from sles. So that's why in previous PRs, this solution was taken. This logic is not introduced in this PR.
I am open to change the solution, as long as there are good reasons, but not convinced by above comment, sorry.  It will need a separate ticket to handle all possible work for it to not break any existing alp test.

Reply:
I know br0 has nothing to do with "default". It is metaphor. My opinion is to keep "default" one if there is one regardless of names (it may not use "default" name). I think keeping it does no harm, but if you think it may has negative impact on you test, you can take reasonable action.

Reply:
Thanks for the suggestion. I will keep this in mind. When it shows the need to keep "default", will create a new ticket for it.